assets/queries/openAPI/callback_object_incorrect_ref/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+	call_back := doc.paths[n][oper].callbacks[c]
+
+	openapi_lib.incorrect_ref(call_back["$ref"], "callbacks")
+
+	result := {
+		"documentId": doc.id,
+		"searchKey": sprintf("openapi.paths.%s.%s.responses.%s.$ref={{%s}}", [n, oper, c, call_back["$ref"]]),
```

Comment:
Shouldn't be somenthing like `openapi.paths.{{%s}}.%s.responses.%s.$ref`?

Reply:
My bad. I thought it was necessary to specify but it is not. Thank you!